greenpithumb/poller.py

```diff
@@ -102,7 +102,7 @@ def _next_poll_time(self):
             self._unix_now(), int(self._poll_interval.total_seconds()))
         if self._last_poll_time and (next_poll_time == self._last_poll_time):
```

Comment:
Doesn't this line not make sense now since ```next_poll_time``` is a POSIX timestamp and ```_last_poll_time``` is a datetime object?